ports/vcpkg-cmake/vcpkg_cmake_configure.cmake

```
@@ -224,9 +224,11 @@ function(vcpkg_cmake_configure)
     if(DEFINED VCPKG_CMAKE_CONFIGURE_OPTIONS_RELEASE)
         vcpkg_list(APPEND arg_OPTIONS_RELEASE ${VCPKG_CMAKE_CONFIGURE_OPTIONS_RELEASE})
     endif()
+    vcpkg_list(APPEND arg_OPTIONS_RELEASE "-DCMAKE_PKG_CONFIG_PC_PATH=${CURRENT_INSTALLED_DIR}/lib/pkgconfig")
     if(DEFINED VCPKG_CMAKE_CONFIGURE_OPTIONS_DEBUG)
         vcpkg_list(APPEND arg_OPTIONS_DEBUG ${VCPKG_CMAKE_CONFIGURE_OPTIONS_DEBUG})
     endif()
+    vcpkg_list(APPEND arg_OPTIONS_DEBUG "-DCMAKE_PKG_CONFIG_PC_PATH=${CURRENT_INSTALLED_DIR}/debug/lib/pkgconfig")
```

- missing `share/pkgconfig`.
- too late: default initialization must be appended before the user-(triplet-)controlled `VCPKG_CMAKE_CONFIGURE_OPTIONS_<config>`.
```suggestion
    vcpkg_list(APPEND arg_OPTIONS_RELEASE "-DCMAKE_PKG_CONFIG_PC_PATH=${CURRENT_INSTALLED_DIR}/lib/pkgconfig;${CURRENT_INSTALLED_DIR}/share/pkgconfig")
    if(DEFINED VCPKG_CMAKE_CONFIGURE_OPTIONS_RELEASE)
        vcpkg_list(APPEND arg_OPTIONS_RELEASE ${VCPKG_CMAKE_CONFIGURE_OPTIONS_RELEASE})
    endif()
    vcpkg_list(APPEND arg_OPTIONS_DEBUG "-DCMAKE_PKG_CONFIG_PC_PATH=${CURRENT_INSTALLED_DIR}/debug/lib/pkgconfig;${CURRENT_INSTALLED_DIR}/share/pkgconfig")
    if(DEFINED VCPKG_CMAKE_CONFIGURE_OPTIONS_DEBUG)
        vcpkg_list(APPEND arg_OPTIONS_DEBUG ${VCPKG_CMAKE_CONFIGURE_OPTIONS_DEBUG})
    endif()
```